Policies/Subscription/Azure_Subscription_SI_Dont_Use_B2C_Tenant/Readme.md

```diff
@@ -0,0 +1,24 @@
+## Azure_Subscription_SI_Dont_Use_B2C_Tenant
+
+### DisplayName 
+[Remove Azure Active Directory B2C tenant(s) in a subscription](../../../Control%20coverage/Feature/SubscriptionCore.md#Azure_Subscription_SI_Dont_Use_B2C_Tenant)
```

Comment:
As policy to cover this  control is partial, let's not push this policy.

Reply:
reverting back